test/e2e/gitops_service_test.go

```diff
@@ -247,3 +261,44 @@ func tearDownArgoCD(t *testing.T) {
 	assertNoError(t, err)
 
 }
+
+func validateMachineConfigUpdates(t *testing.T) {
+	framework.AddToFrameworkScheme(configv1.AddToScheme, &configv1.Image{})
+	ctx := framework.NewContext(t)
+	defer ctx.Cleanup()
+	f := framework.Global
+	path, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	imageYAML := filepath.Join(path, "yamls", "image.yaml")
+	ocPath, err := exec.LookPath("oc")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cmd := exec.Command(ocPath, "apply", "-f", imageYAML)
+	err = cmd.Run()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	time.Sleep(5 * time.Second)
```

Comment:
I thought we agreed not to put sleep here?

Reply:
Yes, including this we have all total 5 `time.Sleep()` in the test suite. This require a separate effort. I have created a high priority issue to track it - https://issues.redhat.com/browse/GITOPS-1034